hack/make-rules/helpers/cache_go_dirs.sh

```diff
@@ -36,7 +36,8 @@ trap "rm -f '${CACHE}'" HUP INT TERM ERR
 # Example:
 #   kfind -type f -name foobar.go
 function kfind() {
-    find .                         \
```

Comment:
Please add a comment why this is the correct path list. Looks like only Makefile.generated_files calls cache_go_dir.sh, luckily.